src/dockerfiles/function/function37.dockerfile

```diff
@@ -8,15 +8,15 @@ USER root
 RUN mkdir -p /app/function/
 
 RUN pip install numpy==1.21.6 \
-pandas==1.3.5 \
-scipy==1.7.3 \
-cloudpickle==2.0.0 \
-pyarrow==7.0.0 \
-boto3==1.18.0 \
-pydantic==1.9.0 \
-scikit_learn==1.0.2 \
-typing_extensions==4.3.0 \
-aqueduct-ml==0.1.10
+    pandas==1.3.5 \
+    scipy==1.7.3 \
+    cloudpickle==2.0.0 \
```

Comment:
@cw75 Let me know if these indents are bad and I can remove. Autoformatting in my IDE did this.

Reply:
@agiron123 Yeah could you remove all the tabs?

Reply:
Sure thing.